blaze/expr/arithmetic.py

```diff
@@ -220,7 +221,7 @@ def scalar_coerce(_, val):
 
 @dispatch(ct.DateTime, _strtypes)
 def scalar_coerce(_, val):
-    return dt_parse(val)
+    return pd.Timestamp(val)
```

Comment:
As an example, I render literal datetime date and timedelta values in `kdbpy` in the way that kdbpy expects them